Command.fs

```diff
@@ -237,14 +244,17 @@ module Queries =
 
     /// Combines the results of each command into a list of all results
     let commandResults cs =
-        // TODO(CaptainHayashi): are sym working variables really results?
-        let primResults prim =
+        let rec primResults prim =
             match prim with
             | Intrinsic (IAssign r) -> [ Expr.Int (r.TypeRec, r.LValue) ]
             | Intrinsic (BAssign r) -> [ Expr.Bool (r.TypeRec, r.LValue) ]
             | Intrinsic (Havoc v) -> [ mkVarExp (mapCTyped Reg v) ]
             | Stored { Results = rs } -> rs
             | SymC _ -> []
+            // TODO(MattWindsor91): is this correct?
+            | PrimBranch (trueBranch = t; falseBranch = f) ->
+                concatMap primResults t
+                @ maybe [] (concatMap primResults) f
```

Comment:
This just collates the results of each branch.  I _think_ this is correct, but am unsure.